drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/analysis/cache/RuleInspectorCache.java

```diff
@@ -65,8 +65,10 @@ public RuleInspectorCache( final ColumnUtilities utils,
 
     public void newColumn( final int columnIndex ) {
 
+        final List<BaseColumn> expandedColumns = model.getExpandedColumns();
+        final BaseColumn baseColumn1 = expandedColumns.get( columnIndex );
```

Comment:
Nothing wrong with being picky.

Looks like at some point I had two base columns. I often use the variables just for easier debugging, I prefer to have them directly in the method/constructor call like they were here before. Going to refactor back to that.